docs/monoDrive_home/Camera.md

```diff
@@ -204,42 +219,55 @@ semantic definition for each pixel value:
 Provides an unsigned 32-bit floating point array where the pixel values 
 represent the distance from the camera in centimeters.
 
-<div class="img_container">
-    <img class='lg_img' src="https://github.com/monoDriveIO/documentation/raw/master/WikiPhotos/camera_depth.png"/>
-</div>
+<div class ='flex'>
+<div class="wide_img">
 
-```
-[
-  {
-     "type": "DepthCamera",
-      "listen_port": 8120,
-      "location": {
-       "x": -800.0,
-       "y": 0.0,
-       "z": 400.0
-     },
-     "rotation": {
-       "pitch": -15.0,
-       "yaw": 0.0,
-       "roll": 0.0
-     },
-     "stream_dimensions": {
-       "x": 512.0,
-       "y": 512.0
-     },
-    "fov": 60.0,
-    "debug_draw": false,
-    "annotation": false,
-    "include_tags": false,
-    "include_obb": false,
-    "cull_partial_frame": false,
-    "far_plane": 5000.0,
-    "desired_tags": [
-      "dynamic"
-    ]
+```json
+{
+    "type": "DepthCamera",
+    "listen_port": 8120,
+    "location": {
+      "x": -800.0,
+      "y": 0.0,
+      "z": 400.0
+    },
+    "rotation": {
+      "pitch": -15.0,
+      "yaw": 0.0,
+      "roll": 0.0
+    },
+    "stream_dimensions": {
+      "x": 512.0,
+      "y": 512.0
+    },
+  "fov": 60.0,
+  "debug_draw": false,
+  "annotation": false,
+  "annotation": {
+        "cull_partial_frame": false,
+        "debug_draw": false,
+        "desired_tags": [],
+        "far_plane": 10000.0,
+        "include_annotation": false,
+        "include_obb": false,
+        "include_tags": false
+    },
+  "include_tags": false,
+  "include_obb": false,
```

Comment:
These are part of the annotation sub json. Not in the outer json.

Reply:
You removed the annotation block, so these still need to be removed when it is added back in.